minetest.conf.example

```diff
@@ -16,6 +16,14 @@
 # Controls
 #
 
+#    If enabled, http links in chat can be middle-clicked or ctrl-left-clicked to open the link in the OS's default web browser.
```

Comment:
you should update settingtypes rather than minetest.conf.example

Reply:
Settingtypes updated, but I had to also leave it in defaultsettings.cpp, else crash on startup.